.env.example

```diff
@@ -82,3 +82,5 @@ ENABLE_SYSTEM_RETURNS_VIEW=true
 GOV_UK_NOTIFY_API_KEY=
 NOTIFICATIONS_BATCH_DELAY=
 NOTIFICATIONS_BATCH_SIZE=
+# The proxy url used for the Notify client
+PROXY=
```

Comment:
The env var we should be using is `http_proxy`.

It's a standard convention (even the use of lowercase,) and all our environments will have this env var.

The previous team used `PROXY` in the import app, and it seemed unnecessary to add a new one or update their code to use `http_proxy`. Hence, I used `PROXY` in my change.

The good news is that it is already available: `config/request.config.js`!

Reply:
bugger